### PR TITLE
[NETBEANS-1074] Module Review groovy.support

### PR DESCRIPTION
**Pull request by junichi11**
- Change the license headers of template files and Bundle.properties
- Add the license headers to META-INF.debugger* files

### CODE REVIEW

File: nbbuild/rat-exclusions.txt [hunk 1]
```diff
@@ -36,6 +36,7 @@ nbbuild/user.build.properties
 **/Makefile-vc
 **/Makefile-vc-wo-crt
 **/package-list
+groovy.support/src/META-INF.debugger*/**
```
Comment (matthiasblaesing) [inline nbbuild/rat-exclusions.txt]: This is IMHO not needed. If I'm not mistaken the code, that reads the files is `api.debugger/src/org/netbeans/api/debugger/Lookup.java` and there comment sign `#` is explicitly mentioned, so I would add the ALv2 header directly. There are also debugger related files `spi.debugger.ui/build/classes/META-INF/debugger/ThreadsView/org.netbeans.spi.viewmodel.ColumnModel` that show the header.

So I would not add it to the exlusion list, but add the header.

Comment (junichi11) [reply nbbuild/rat-exclusions.txt]: I'll add the license header later. Thanks!

**Comment by junichi11** [comment]
Fixed.

**Comment by geertjanw** [comment]
Thanks, merging.

**Pull request merged**